aidial_adapter_openai/utils/remove_none.py

```diff
@@ -0,0 +1,11 @@
+from typing import Any, Dict, Optional
+
+
+def remove_none(d: Dict[str, Optional[Any]]) -> Dict[str, Any]:
+    """
```

Comment:
IMO, 6 lines of comments for 1 line of actual code is an overkill.

Noone ever is going to read this comment from start to finish. 
One would jump straight away to the line of code.

Reply:
Currently it doesn't make sense, you're right

But it could make sense to add such `reStructuredText` (or maybe google/numpy type of comments) to each function, so at some point we could generate automatic documentation using some tool like `Sphinx`

Reply:
I agree, but we don't have such plans yet. 

Also, automatic documentation is a good idea for libraries, but this adapter isn't a library but an application on its own.